1/hangman.py

```diff
@@ -0,0 +1,45 @@
+# Problem: the game is currently case-sensitive. A != a
+import getpass
+
+# Split the keyword into characters and put them into an array
+def split(keyword):
+    return [char for char in keyword]
+
+# Identify the number of guesses allowed
+def minimum(answer):
+    if len(answer) < 5:
+        return 5
+    else:
+        return len(answer) + 2
+
+# Check if the guessed letter matches something in the array, if so, replace the empty space in box
+def check_answer(guess):
+    for index, letter in enumerate(answer):
+        if letter == guess:
+            box[index] = guess

```

Comment:
ok this works but is a pattern we want to avoid. 

either pass in box as an argument or dont use box at all. The reason is your function is now accessing a variable that's outside its scope and can easily create bugs